hw3/css/stylehw3.css

```diff
@@ -0,0 +1,477 @@
+	left: 0;
+	bottom: 0;
+	padding: 7px;
+	font-size: 16px;
+	line-height: 20px;
+	cursor: pointer;
+}
+.services__box1__title::before {
+	content: url(../img/browser.png);
+	display: block;
+	margin-right: 3px;
+}
+.services__box2__title::before {
+	content: url(../img/e-commerce.png);
+	display: block;
+	margin-right: 3px;
+}
+.services__box3__title::before {
+	content: url(../img/mouse.png);
+	display: block;
+	margin-right: 3px;
+}
+/*PROJECTS*/
+.projects {
+	display: flex;
+	justify-content: space-between;
+	flex-wrap: wrap;
+	line-height: 20px;
+}
+.projects__box1,
+.projects__box2,
+.projects__box3 {
+	width: 29%; 
+	margin-top: 40px;
+}
+.projects__box1__example,
+.projects__box2__example,
+.projects__box3__example {
+	position: relative;
+	background:#fff;
+	border-radius: 3px;
+	border: 7px solid #e2e0e0;
+	height: 150px;
+	margin-bottom: 10px;
+	background-size: cover;
+}
+.projects__box1__example{
+	background-image: url(../img/terrion.png);
+}
+.projects__box2__example{
+	background-image: url(../img/psd.png);
+}
+.projects__box3__example{
+	background-image: url(../img/layer-7.png);
+}
+.projects__box1__example:before, .projects__box1__example:after,
+.projects__box2__example:before, .projects__box2__example:after,
+.projects__box3__example:before, .projects__box3__example:after
+{
+  z-index: -1;
+  position: absolute;
+  content: "";
+  bottom: 5px;
+  left: 10px;
+  width: 50%;
+  top: 50%;
+  max-width:130px;
+  background: #777;
+  -webkit-box-shadow: 0 15px 10px #777;
+  -moz-box-shadow: 0 15px 10px #777;
+  box-shadow: 0 15px 10px #777;
+  -webkit-transform: rotate(3deg);
+  -moz-transform: rotate(3deg);
+  -o-transform: rotate(3deg);
+  -ms-transform: rotate(3deg);
+  transform: rotate(3deg);
+}
+.projects__box1__example:after,
+.projects__box2__example:after,
+.projects__box3__example:after 
+{
+  -webkit-transform: rotate(-3deg);
+  -moz-transform: rotate(-3deg);
+  -o-transform: rotate(-3deg);
+  -ms-transform: rotate(-3deg);
+  transform: rotate(-3deg);
+  right: 10px;
+  left: auto;
+}
+.projects__box1__title,
+.projects__box2__title,
+.projects__box3__title {
+	text-transform: capitalize;	
+	font-size: 18px;
+	font-weight: 700;
+}
+.projects__box1__text,
+.projects__box2__text,
+.projects__box3__text {
+	margin-top: 10px;
+}
+/*CONTACT*/
+.information {
+	background-image: url(../img/contact.png);
+	background-repeat: repeat-x;
+	background-size: cover;
+}
+.information__content {
+	display: flex;
+	justify-content: space-between;
+	flex-wrap: wrap;
+	width: 80%;
+	margin: 0 auto;
+	padding-bottom: 20px;
+	color: #fff;
+	line-height: 20px;
+}
+.information__content__box1,
+.information__content__box2,
+.information__content__box3 {
+	width: 29%;
+	padding-top: 20px;
+}
+.information__content__box1__title,
+.information__content__box2__title,
+.information__content__box3__title {
+	line-height: 30px;
+	font-size: 24px;
+	padding: 10px 0;
+	border-bottom: 5px double black;
+}
+.information__content__box1__text {
```

Comment:
remove those fucking classes with numbers. Nobody would ever review this MR with such stupid amount of the same classes.